org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/BaseValidator.java

```diff
@@ -1,84 +1,9 @@
 package org.hl7.fhir.validation;
 
-import static org.apache.commons.lang3.StringUtils.isBlank;
-
-import java.text.DateFormat;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.HashMap;
-
-/*
-  Copyright (c) 2011+, HL7, Inc.

```

Comment:
The copyright notice shouldn't be deleted.

Reply:
Dunno how that happened.
Fixed.